features/security-monitoring.md

```diff
@@ -0,0 +1,26 @@
+# Security Monitoring
+The Private Packagist security monitoring feature analyzes the dependencies of your private packages for known
+security vulnerabilities using various databases like [sensiolabs/security-advisories](https://github.com/FriendsOfPHP/security-advisories).
```

Comment:
I think this page should document the exact list of sources used rather than an example?

Reply:
That makes sense. Update it.